lib/modules/manager/pipenv/__fixtures__/Pipfile4

```diff
@@ -11,6 +11,9 @@ requests = { extras = ['socks'] }
 records = '>0.5.0'
 django = { git = 'https://github.com/django/django.git', ref = '1.11.4', editable = true }
 flask = { git = 'https://github.com/pallets/flask.git' }
+python-gitlab = { git = 'https://gitlab.com/python-gitlab/python-gitlab', ref = '1.8.0', editable = true }
```

Comment:
```suggestion
```

the django tests same

Reply:
I added the gitlab one which uses `git-tags` instead of the `github-tags` datasource. If I remove it I think the test coverage will go down, as all other examples in Pipfile4 use github.com

Reply:
ok, didn't notice. 🤗